Application/EdFi.Ods.Common/Security/EdFiOdsApiClaimTypes.cs

```diff
@@ -19,5 +19,10 @@ public static class EdFiOdsApiClaimTypes
         /// The name of an assigned Ed-Fi API profile.
         /// </summary>
         public const string Profile = @"http://ed-fi.org/claims/profile";
+
+        /// <summary>
+        /// The name of the claim set assigned to the Ed-Fi ODS API client.
+        /// </summary>
+        public const string ClaimSetName = @"http:// ed-fi.org/claims/claimSetName";
```

Comment:
should this be `@"http://ed-fi.org/claims/claimSetName";` instead of `@"http:// ed-fi.org/claims/claimSetName";`

Reply:
I like to include random spaces in my URLs, primarily to keep you on your toes.  (Will fix.)